api/server.py

```diff
@@ -72,9 +67,16 @@ async def exception_handler(_: Request, e: Exception):
         code = determine_code_from_error(e)
         name = str(type(e).__name__)
 
+    base_params = {'name': name, 'description': str(e)}
+
+    if add_stacktrace:
+        st = traceback.format_exc()
+        logger.error(traceback.format_exc())
+        base_params['stacktrace'] = st
+
```

Comment:
I reordered this, so the stacktrace appears at the bottom of the json response, dictionary insert order matters here.

Reply:
Cool! 
Could you write `logger.error(st)` in the line 74?